### ISSUE TITLE
URL Router 3.0 not working with Sym 2.4+

### ISSUE THREAD

**Issue by stuartgpalmer** [symphonists/url_router#20]
I have consistently used this route to shortcut URLs such as:

> example.com/page/about/

to 

> example.com/about/

![alt text](http://www.getsymphony.com/global-asset-download/file/107748/Screen%20Shot%202014-09-03%20at%2014.57.04.png)

The page "page" accepts the parameter "$page". This works with Sym 2.3.x and URL Router 2.0.2.

On Sym versions 2.4 and 2.5.1 and URL Router 3.0 this no longer seems to work. Is there anything which has changed on either release which would cause this to fail? Thanks

I don't see any errors, I am simple sent to the `index` page rather than the `page` page. Its as if  Debug data looks like this:

```
    <root-page>home</root-page>
    <current-page>home</current-page>
    <current-page-id>1</current-page-id>
    <current-path>/who-we-are</current-path>
```

The index page isn't setup to accept URL parameters. If I disable the extension I then get a 404.

Nothing is included in the logs.

I'm happy to provide sum login details if anyone would be happy to have a look.

**Comment by brendo** [comment]
Check out `integration`, I think I've fixed this :)

**Comment by stuartgpalmer** [comment]
This does indeed fix the routing issue. I notice that the "Force re-route even if page exists" checkbox has gone now though.

**Comment by brendo** [comment]
@stuartgpalmer Just checking, `integration` is all good to merge into `master` now?

**Comment by stuartgpalmer** [comment]
Hi Brendo

Yes, it's all working fine with 2.5 now. Thanks. 

S

Stuart Palmer
Eyes-Down Ltd
The Lodge Studio, Duncannon Lane,
Stoke Gabriel, Devon, TQ9 6QY

Tel: 01803 500099 (office), 07944 016360 (mobile)
Skype: stuartgpalmer
LinkedIn: /in/stuartgpalmer/
Website: www.eyes-down.net 

View Stuart's profile

**Comment by brendo** [comment]
Thanks Stuart, I've released 3.1 :)

**Comment by stuartgpalmer** [comment]
Great. Thanks Brendo
